File: openspec-dotnet/Model/Employee.cs
namespace EmployeeApi.Model;

// Plain POCO — no EF Core mapping/attributes yet, intentionally
public class Employee
{
    public int? Id { get; set; }
    public string? FirstName { get; set; }
    public string? LastName { get; set; }
    public string? Email { get; set; }
    public string? Department { get; set; }
}

File: openspec-py/app/model/employee.py
from dataclasses import dataclass
from typing import Optional


# Plain domain model — no SQLAlchemy mapping yet, intentionally
@dataclass
class Employee:
    id: Optional[int] = None
    first_name: Optional[str] = None
    last_name: Optional[str] = None
    email: Optional[str] = None
    department: Optional[str] = None

